setup.cfg

```diff
@@ -28,7 +28,7 @@ keywords = colcon
 install_requires =
   catkin_pkg
   colcon-cmake>=0.2.0
-  colcon-core>=0.3.1
+  colcon-core>=0.3.10
```

Comment:
Please also update the version in the Debian patch file: https://github.com/colcon/colcon-ros/blob/9bdb0427f1e07f41a30a5dd6f3e037393af0de6f/debian/patches/setup.cfg.patch#L22